README.md

```diff
@@ -80,6 +80,18 @@ If you're comfortable with the command line, the Azure CLI provides the `deploym
       --name $deploymentName \
       --template-file ./mockSpacestation.json
     ```
+    Note: Azure Portal deployment supports overriding the default Groundstation and Spacestation location(s) via the UI. When using the CLI, you can override the default location(s) using a `--parameters` argument to override the default `groundstationLocation` and/or `spacestationLocation` parameter(s). For example:
+    ```bash
+    groundstationLocation="usgovvirginia"
+    spacestationLocation="usgovarizona"
+
+    az deployment group create \
+      --resource-group $resourceGroupName \
+      --name $deploymentName \
+      --parameters groundstationLocation=$groundstationLocation \
+      --parameters spacestationLocation=$spacestationLocation \
+      --template-file ./mockSpacestation.json    
+    ```
```

Comment:
Got some markdown linting warnings. How does this look?

```suggestion

    Note: Azure Portal deployment supports overriding the default Groundstation and Spacestation location(s) via the UI. When using the CLI, you can override the default location(s) using a `--parameters` argument to override the default `groundstationLocation` and/or `spacestationLocation` parameter(s). For example:

    ```bash
    groundstationLocation="usgovvirginia"
    spacestationLocation="usgovarizona"

    az deployment group create \
      --resource-group $resourceGroupName \
      --name $deploymentName \
      --parameters groundstationLocation=$groundstationLocation \
      --parameters spacestationLocation=$spacestationLocation \
      --template-file ./mockSpacestation.json    
    ```
```